python_modules/dagster/dagster_tests/core_tests/run_coordinator_tests/test_queued_run_coordinator.py

```diff
@@ -151,6 +151,17 @@ def test_submit_run(self, instance, coordinator, workspace, remote_job):
         stored_run = instance.get_run_by_id(run.run_id)
         assert stored_run.status == DagsterRunStatus.QUEUED
 
+        events = list(
+            instance.get_records_for_run(
+                run.run_id, of_type=DagsterEventType.PIPELINE_ENQUEUED
```

Comment:
Haha sneaky - so I guess this is our assertion that the enum name change is fine?